libgadget/timestep.c

```diff
@@ -686,36 +686,85 @@ inttime_t find_next_kick(inttime_t Ti_Current, int minTimeBin)
     return Ti_Current + dti_from_timebin(minTimeBin);
 }
 
+static void print_timebin_statistics(int NumCurrentTiStep, int * TimeBinCountType);
+
 /* mark the bins that will be active before the next kick*/
-int rebuild_activelist(inttime_t Ti_Current)
+int rebuild_activelist(inttime_t Ti_Current, int NumCurrentTiStep)
```

Comment:
NumCurrentTiStep is only used in the print. Is there a way to reformat the print to get rid of it, or moving it to All (since it is globally synced anyways?)

Reply:
We could; it isn't really used anywhere except printing the logs though. I think I prefer to keep it local, it isn't too much hassle to pass it into the two functions that use it.